frontend/src/app/confirm.service.spec.ts

```diff
@@ -80,3 +81,18 @@ describe('ConfirmService and its modal compoent', () => {
     expect(document.querySelector('.modal-content')).toBeFalsy();
   });
 });
+
+@Component({
+  template: ''
+})
+class TestComponent {
+}
+
+@NgModule({
+  imports: [RouterTestingModule, NgbModule.forRoot()],
+  declarations: [ConfirmModalContentComponent, TestComponent],
+  providers: [ConfirmService],
+  entryComponents: [ConfirmModalContentComponent]
+})
+class TestModule {
```

Comment:
After reviewing the rest I see that you used this pattern in test that have several `configureTestingModule`. It's not necessary here but that's ok.